common/common/segments.py

```diff
@@ -222,7 +222,14 @@ def best_segments_by_start(hour):
 	segment_paths.sort()
 	# note we only parse them as we need them, which is unlikely to save us much time overall
 	# but is easy enough to do, so we might as well.
```

Comment:
with this change, this comment is no longer true (you're parsing them all now, not lazily). I don't think the laziness was actually valuable, but we should delete the comment.